db/migrate/20170502172444_create_items.rb

```diff
@@ -0,0 +1,13 @@
+class CreateItems < ActiveRecord::Migration[5.0]
+  def change
+    create_table :items do |t|
+      t.string :name
+      t.text :description
+      t.integer :unit_price
+      t.datetime :created_at
+      t.datetime :updated_at
+
+      t.timestamps
```

Comment:
Think it would be clearer if we render the timestamps as null: true since we already have created_at and updated_at in our model.